packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -2076,6 +2076,13 @@ export interface TabsMain {
     $closeGroup(groupIds: number[], preserveFocus?: boolean): Promise<boolean>;
 }
 
+export interface TelemetryMain {
+
+}
+
+export interface TelemetryExt {
+}
```

Comment:
minor: stick with newline or no newline for empty interfaces, the same will also apply to `TelemetryLoggerOptions`.

Reply:
I removed the empty line in TelemetryMain, as for TelemetryLoggerOptions.